LearnositySDK/Request/Init.cs

```diff
@@ -536,6 +536,12 @@ public void addTelemetryData()
             if (this.isTelemetryEnabled())
             {
                 JsonObject meta;
+
+                if (this.requestPacket == null)
+                {
+                    this.requestPacket = new JsonObject();
+                }
+
```

Comment:
What are your thoughts on moving this higher up in the call stack, like in `Initialize`? Doing so might make it less likely that we have to duplicate this code to fix/prevent similar issues in the future (though maybe it would cause issues for the outputs? Good to check).

Reply:
I've moved this up into Initialize as suggested. Doesn't seem to have caused any issues by doing this.